### ISSUE TITLE
[Meta] Can we open GitHub Discussions "forum" on this repo ?

### ISSUE THREAD

**Issue by AaronNGray** [w3c-social/activitypump#426]
Can we please enable the GitHub Discussions "forum" on this repo in the settings ?

**Comment by nightpool** [comment]
What would be the benefit of this over using the mailing list?

**Comment by AaronNGray** [comment]
@nightpool Its a separate more formal channel thats associated with this repo and the issues, that is less formal than issues, and where ideas and queries do not get lost in linear narrative and peoples inboxes. And will free the SWICG mailing list for more general discussions which it seems more biased towards anyway.

**Comment by TallTed** [comment]
Note that GitHub's "Discussions" are not really discussions. They are a reinvention of the Q&A sites like StackOverflow. They do not thread properly; they associate multiple "answers" to the initial comment (the "question"), and multiple "comments" may be made on each "answer", but these "comments" are single-level. They are not easily caught-up on, as there is no "show me all comments since my last visit" sort of view. It is trivially easy to overlook newer "comments" on older "answers". I strongly advise *against* using GitHub's "Discussions" for anything other than a Q&A forum, and almost as strongly advise using a more actively evolving web-based reinvention of the threaded discussions of NetNews/Usenet/NNTP (by which I do not mean GoogleGroups).

**Comment by AaronNGray** [comment]
NNTP would be nice, Google is dropping Google Groups a.k.a. UseNet apparently. I think a Q&A style discussions would still be fine for our purposes.

**Comment by ThisIsMissEm** [comment]
The idea from the Social Web CG call yesterday was to give non-SWICG people a place to ask questions, such that we can further build out the Primer and Errata. The idea for using GitHub Discussions, instead of the mailing list or a separate forums software was because the mailing list is more spec contributors & standards folks, and isn't really the most accessible option. As for separate forum software, it'd require someone to deploy and maintain it, and we already have socialhub, but it doesn't tend to lead back into the Social Web CG's activities.

**Comment by AaronNGray** [comment]
@ThisIsMissEm There is a separate forum based ActivityPub discussion group, but its not for the spec itself, more just for implementors. I suppose we have to use issues here rather than discussion ;)
https://socialhub.activitypub.rocks/c/activitypub/5

**Comment by nightpool** [comment]
I could open up another category at Socialhub (or enforce the existing
categories more strictly) if people think it would help discussion

**Comment by AaronNGray** [comment]
@nightpool The idea was to have something associated with this repo. I think something separate would probably just get ignored and forgotten about over time.

**Comment by ThisIsMissEm** [comment]
That or we add a issue template to the repository to direct questions to that forum, e.g., here's Mastodon's: https://github.com/mastodon/mastodon/blob/main/.github/ISSUE_TEMPLATE/config.yml

**Comment by dmitrizagidulin** [comment]
I agree with @nightpool  -- I'd much rather see the Q&A use a combination of Github templates (using something like the template @ThisIsMissEm pointed to above), PLUS the SocialHub subforum (for those who prefer a more informal venue).
Definitely reluctant to use Discussions, for the reasons @TallTed mentioned.

**Comment by evanp** [comment]
I would like to suggest that with both a mailing list and a Web forum at SocialHub, we are covered for a couple of important modalities of communication. We also have the #ActivityPub and #SocialCG hashtags on the fediverse itself. So, I am -1 on adding yet another official form of communication -- although I think 1000 flowers should bloom in all the different areas people want to talk.

First, I'd like to update the README for this repo to include links on how to join SocialCG or review the mailing list; on how to connect to SocialHub and in particular the ActivityPub discussions there; and link to other resources like the Primer or the hashtags.

Then, I agree with @ThisIsMissEm that we can include that information in the flow for creating a new issue, if that's the main entry point.

**Comment by evanp** [comment]
This ticket is over a year old; I don't see a lot of momentum for opening up the Discussions forums, especially since we have other ways to discuss the spec. I'm going to drop the Needs Group Input tag and change to waiting for commenter. @AaronNGray it would be great to hear your thoughts at this point.

**Comment by evanp** [comment]
Given there hasn't been action since the last update, I'm going to close this ticket. If needed, it can be reopened to reopen the issue.
